src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+
+  const basicDiscount = 20;
+  const basicDiscountDays = 3;
+
+  const advancedDiscount = 50;
+  const advancedDiscountDays = 7;
+
+  const rentCost = dayCost * days;
+
+  if (days >= advancedDiscountDays) {
+    return rentCost - advancedDiscount;
+  }
+
+  if (days >= basicDiscountDays) {
+    return rentCost - basicDiscount;
+  }
+
+  return rentCost;
```

Comment:
too big solution

Reply:
Here is a short one:

`const calculateRentalCost = d => d >= 7 ? 40 * d - 50  : d >= 3 ? 40 * d - 20 : 40 * d;`

Reply:
:harold: